ninja-core/src/main/java/ninja/template/TemplateEngineFreemarkerExceptionHandler.java

```diff
@@ -53,38 +52,9 @@ public TemplateEngineFreemarkerExceptionHandler(Logger logger,
 
     public void handleTemplateException(TemplateException te,
```

Comment:
Not sure - but would it make sense to replace this templateerror handler with something more generic? If we have a mechanism in place that catches exceptions bubbling up - why should it not display a nicely formatted error there - with the added benefit that we can see the all kinds of exeptions in the browser while developing (not just templating expceptions)... just an idea...